todo_app/tests/integration/test_client.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pytest
+import requests
+from todo_app import app
+
+@pytest.fixture
+def client():
+    test_app = app.create_app('.env.test')
+
+    with test_app.test_client() as client:
+        yield client
+
+class TestClient:
+    def test_index_pages(self, monkeypatch, client):
+        monkeypatch.setattr(requests, 'get', get_lists_stub)
+        response = client.get('/')
+
+        assert "Test card" in response.data.decode()
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+def get_lists_stub(url, params):
```

Comment:
I'm not entirely sure why, but to get the test running locally I needed to add a default value to params, i.e. `params=None`